tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -0,0 +1,1180 @@
+      await expect(page
+        .getByTestId('granule-results-actions__download-all-button')
+        .locator('.button__badge')).toHaveText(commafy(cmrHits))
+    })
+  })
+
+  test.describe('When the path is loaded with a collection with an active subscription', () => {
+    test('loads with the subscription indicator active', async ({ page, context }) => {
+      const conceptId = 'C1214470488-ASF'
+      const cmrHits = 229
+
+      await login(context)
+
+      await page.route('**/search/collections.json', async (route) => {
+        const request = route.request()
+        const body = request.postData()
+        expect(body).toEqual('has_granules_or_cwic=true&include_facets=v2&include_granule_counts=true&include_has_granules=true&include_tags=edsc.*,opensearch.granule.osdd&page_num=1&page_size=20&point[]=-77.04119,38.80585&sort_key[]=has_granules_or_cwic&sort_key[]=-score')
+
+        await route.fulfill({
+          body: JSON.stringify(commonBody),
+          headers: commonHeaders
+        })
+      })
+
+      await page.route('**/search/granules.json', async (route) => {
+        const request = route.request()
+        const body = request.postData()
+        expect(body).toEqual('echo_collection_id=C1214470488-ASF&page_num=1&page_size=20&point[]=-77.04119,38.80585&sort_key=-start_date')
+
+        await route.fulfill({
+          body: JSON.stringify(subscriptionGranulesBody),
+          headers: {
+            ...commonGranulesHeaders,
+            'access-control-expose-headers': 'cmr-hits',
+            'cmr-hits': cmrHits
+          }
+        })
+      })
+
+      await page.route('**/search/granules/timeline', async (route) => {
+        const request = route.request()
+        const body = request.postData()
+        expect(body).toEqual('end_date=2023-12-01T00:00:00.000Z&interval=day&start_date=2018-12-01T00:00:00.000Z&concept_id[]=C1214470488-ASF')
+
+        await route.fulfill({
+          body: JSON.stringify(subscriptionTimelineBody),
+          headers: subscriptionTimelineHeaders
+        })
+      })
+
+      await page.route('**/graphql', async (route) => {
+        const request = route.request()
+        const body = JSON.parse(request.postData())
+
+        if (body.data.query === graphQlGetSubscriptionsQuery) {
+          await route.fulfill({
+            body: JSON.stringify({
+              data: {
+                subscriptions: {
+                  items: []
+                }
+              }
+            }),
+            headers: subscriptionGraphQlHeaders
+          })
+        } else if (body.data.query === JSON.parse(graphQlGetCollection(conceptId)).query) {
+          await route.fulfill({
+            body: JSON.stringify(subscriptionGraphQlBody),
+            headers: graphQlHeaders
+          })
+        }
+      })
+
+      await page.goto('/search/granules?p=C1214470488-ASF&pg[0][gsk]=-start_date&sp[0]=-77.04119,38.80585')
+
```

Comment:
```suggestion
 // Ensure the correct number of results were loaded
      testResultsSize(page, cmrHits)
```
the cypress test had this assertion for this test